src/tabs/tabs.scss

```diff
@@ -112,6 +111,32 @@
   }
 }
 
+@mixin iui-tabs-horizontal {
+  flex-direction: column;
+
+  .iui-tabs {
+    display: flex;
+  }
+}
+
+@mixin iui-tabs-vertical {
+  flex-direction: row;
+
+  .iui-tabs {
+    display: inline-flex;
```

Comment:
It seems like these two lines can be omited and it will look the same.

Reply:
Nice catch, you are correct.  Removed.